src/Scripting/CSharpTest/ScriptTests.cs

```diff
@@ -292,6 +302,13 @@ public async Task TestRunScriptWithGlobals()
             Assert.Equal(3, state.ReturnValue);
         }
 
+        [Fact]
+        public async Task TestRunScriptWithBaseGlobals()
```

Comment:
>public [](start = 8, length = 6)

Consider adding: `[WorkItem(30947, "[https://github.com/dotnet/roslyn/issues/30947](https://github.com/dotnet/roslyn/issues/30947)")]`.

Reply:
Sorry, I pasted the link incorrectly. The attribute should be: `[WorkItem(30947, "https://github.com/dotnet/roslyn/issues/30947")]`